docs/UserGuide.md

```diff
@@ -123,11 +124,17 @@ Alias: `find`
 * Keywords can overlap. e.g. `samm my` will match `Sammy`
 * Fosters must match all keywords (i.e. `AND` search).
   e.g. `Hans Bo` will return `Hansbo Grahm`, but not `Hans Duo`
+* Symbols between keywords or sections will combine them according to the function of the symbol.
+  `|` is `or`, `&` is `and` (low precedence), and ` ` (space) is `and` (high precedence).
+  e.g. `a & b | c d` is the same as `a & (b | (c & d))`
+* Use double quotes `"` for exact, case-sensitive, word-level match. 
+  e.g. `"Tom"` matches "Tom", but not "Tommy"
 
 Examples:
 * `list` lists all fosterers in the address book
 * `list john doe` matches "John Doe", "Doe John", "Johnny Doe", and "Mary" who lives on "John Doe Street"
-* `list john john doe` is redundant and gives the same result as `list john doe`
+* `find john john doe` is redundant and gives the same result as `find john doe`
+* `list "John" | zam & doe` matches "John Doe" and "Doe Shazam", but not "John Grahm"
 
 Expected output (success):
```

Comment:
I think this is not the current success output if im not wrong, but if you are planning to change it to this then can just leave it just such! same for failure expected output

Reply:
I changed the doc to reflect the actual output, thanks!